docs/tutorials/slurm/compute_canada/compute_canada.rst

```diff
@@ -23,6 +22,10 @@ or
 .. code-block::
 
    $ rsync -avPhz input_clar <username>@niagara.computecanada.edu:/scratch/<username>/.
+
+or
+
+For large datasets, or if you desire a GUI, you can setup a Globus transfer server by following instructions on the [Digital Research Alliance of Canada wiki](https://docs.alliancecan.ca/wiki/Transferring_data "Compute Canada Wiki").  
```

Comment:
Could you pleace replace this paragraph with the one below using correct `rst` syntax for links instead of Markdown as well as slightly different wording:

"Alternatively, for large datasets, or if you desire a GUI, you can setup a Globus transfer server by following instructions on the `Digital Research Alliance of Canada wiki <https://docs.alliancecan.ca/wiki/Transferring_data>`_."